pkg/scraper/scrapers.go

```diff
@@ -509,3 +507,36 @@ func (c Cache) ScrapeMovieURL(url string) (*models.ScrapedMovie, error) {
 
 	return nil, nil
 }
+
+func postProcessTags(tqb models.TagReader, scrapedTags []*models.ScrapedSceneTag) ([]*models.ScrapedSceneTag, error) {
+	var ret []*models.ScrapedSceneTag
+
+	excludePatterns := stash_config.GetInstance().GetScraperExcludeTagPatterns()
+	var excludeRegexps []*regexp.Regexp
+
+	for _, excludePattern := range excludePatterns {
+		reg, err := regexp.Compile(strings.ToLower(excludePattern))
+		if err != nil {
+			logger.Errorf("Invalid tag exclusion pattern :%v", err)
+		} else {
+			excludeRegexps = append(excludeRegexps, reg)
+		}
+	}
+
+ScrapeTag:
+	for _, t := range scrapedTags {
+		for _, reg := range excludeRegexps {
+			if reg.MatchString(t.Name) {
```

Comment:
It should be
```golang
                       if reg.MatchString(strings.ToLower(t.Name)) {
```
Since you convert the regexps to lower the tag names should also be converted to lower (the same is done in the file exclusion code). This would make the matches case insensitive (as they should be imo) without having the user add the `(?i)` in the regex. Only making the regexp lower case doesnt make sense.

I would also add a `logger.Debugf` message with the tag that was skipped due to being blacklisted before the `continue`

Reply:
Not sure how I missed the `ToLower` in file exclussions. Anyway, it's added now.

I did add logging as well, but instead of logging every tag I collect them and add a single log entry containing all ignored tags. This message is also logged as info, instead of debug. Because file exclusion also logs as info I saw.

Reply:
Yeah everything seems ok now.